packages/docs/site/docs/08-query-api/01-index.md

```diff
@@ -24,15 +24,15 @@ You can go ahead and try it out. The Playground will automatically install the t
 | `url`          | `/wp-admin/`  | Load the specified initial page displaying WordPress                                                                                                                                                                                                                                                                                                           |
 | `mode`         | `seamless`    | Displays WordPress on a full-page or wraps it in a browser UI                                                                                                                                                                                                                                                                                                  |
 | `lazy`         |               | Defer loading the Playground assets until someone clicks on the "Run" button                                                                                                                                                                                                                                                                                   |
 | `login`        | `1`           | Logs the user in as an admin                                                                                                                                                                                                                                                                                                                                   |
 | `gutenberg-pr` |               | Loads the specified Gutenberg Pull Request                                                                                                                                                                                                                                                                                                                     |
-| `storage`      |               | Selects the storage for Playground: `none` gets erased on page refresh, `browser` is stored in the browser, and `device` is stored in the selected directory on a device. The last two protect the user from accidentally losing their work upon page refresh.                                                                            |
+| `storage`      |               | Selects the storage for Playground: `temporary` gets erased on page refresh, `browser` is stored in the browser, and `opfs-host` is stored in the selected directory on a device. The last two protect the user from accidentally losing their work upon page refresh.                                                                                         |
```

Comment:
```suggestion
| `storage`      |               | Selects the storage for Playground: `none` gets erased on page refresh, `browser` is stored in the browser, and `device` is stored in the selected directory on a device. The last two protect the user from accidentally losing their work upon page refresh.                                                                                         |
```

Reply:
@ajitbohra I saw you rebased but this issue is still present. If you just commit this suggestion I'll go ahead and merge your PR. Thank you!